packages/debugger/src/tokens.ts

```diff
@@ -672,12 +770,12 @@ export namespace IDebugger {
       /**
        * Signal emitted when the current frame has changed.
        */
-      readonly currentFrameChanged: ISignal<this, IDebugger.IStackFrame>;
+      readonly currentFrameChanged: ISignal<this, IDebugger.IStackFrame | null>;
```

Comment:
Let's tag as `api-change` since there are a couple of signature changes in the public interfaces.